### ISSUE TITLE
Define payload in terms of role and other payload issues

### ISSUE THREAD

**Issue by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#245]
A camera or a gun can be a payload and is this would entail multiple parenthood. In order to avoid that, please define a payload role and then define payload as a material entity that has that role using equivalent class. Combat payload should also be similarly defined.

The other subclasses are defining amounts relative to an aircraft, but there is no connection to the class aircraft. I'm not quite sure what to define them in terms of. One the one hand these might be considered information artifacts about an aircraft. They would be value specifications that describe, in that case, with the units being either weight or volume units. 

Or perhaps they are defined as dispositions, with the realizations having participants with aircraft and payloads with certain (weight or space) qualities.

What is not clear to me is whether these are intended to be used as specifications, as in, the maximum payload x aircraft can take is 2 tons, or whether they refer to particular payloads, in which case one would expect an instance of maximum payload to instantiate normal payload at some point, as expendables are used, and for it to cease to exist once all the expendables are gone. I *think* it is the specification sense that is meant but someone should confirm that. It is on the former interpretation that I base my suggested representations.

As written it would seem that Normal Aircraft Payload is a subclass of Combat Payload, and probably the label should be changed to reflect that.

I'm also not sure that the Maximum, Minimum, Normal payloads should be defined in terms of expendables, as they are now, as the weight and space specifications would seem to equally apply to non-expendable cargo (like cameras and equipment).

In the specifications I am working with, there is a corresponding role for the vehicle, a platform role and these are mutually dependent, i.e. payloads are intended to be carried by platforms, and platforms are the sort of thing that can carry payloads.

**Comment by cameronmore** [comment]
Is a payload a role? It seems like payload is really about the participation in a process as you pointed out, but more like a measurement that something was or is designated to be a participant in some process, not necessarily a role itself. If I ask what the organizational context of a payload role would be, the answer sounds like participation in a process, and not some actual organization except for the people who are agents in that process, but I think that stretches organizational contacts would be, since again that would be about whether a process happened or not, and not necessarily a set of social realizable circumstances?

**Comment by alanruttenberg** [comment]
No, I do think they are a role. I work with cameras as payload. They are payload whether the aircraft is sitting on the ground or transporting them. Characteristics are associated with them. Even though they may have a function, a variety of different artifacts are grouped into what are called payload. Basically, it's a payload if someone says it's a payload. You even reflect it in your language "something was or is designated to be a participant in some process". The "is designated to be" are exactly how realizables come into existence.  Something is conferred on something, and the thing conferred is realized in a process. The processes you refer to are their realization, which is roughly, in addition to realizing the function of devices, realize their roles as payload. 

>Several types of Payloads hosted on Platforms provide distinct functions to achieve system and mission objectives. One subset of Payload types are sensors that gather Geographical based Intelligence data on an image scene (see MISB Motion Imagery Handbook [3] for background on Sensors and Scene). A Geo-Intelligence Sensor is a device which measures active or passive Scene energy with a defined correspondence between the sensed data and the Scene. 
>While Payload devices share common attributes, each Payload device also defines specific metadata for their phenomenology’s and methods of data collection. This standard includes the model for frame imagers (visible light and infrared), and Laser Range Finder (LRF); future revisions will include models for other Payload devices.

In our language, the payload is a role and when they talk about "specific metadata" they are talking about characterizing the function and operation of the various devices that can as a group be called payload.

Remember that I'm distinguishing the quantified payload terms differently than "Combat payload" or "Imaging payload". Those I think aren't roles.

There is also 'Payload Capacity' to pay attention to.

**Comment by cameronmore** [comment]
That makes sense, and the act of designation seems to be doing far more of the work. Alongside other things like component role and infrastructure role, I think the role makes sense.

Payload capacity is definitely about the disposition of the craft to carry the payload, whatever it may be, and still realize its disposition.

**Comment by alanruttenberg** [comment]
Yes, and I think the sense of those payload quantifications as dispositions I was thinking of. In that light, those might belong under payload capacity.
Expendable would be another role, mentioned in Normal and Maximum Aircraft Payload

**Comment by cameronmore** [comment]
Jon Vajda recently commented on the [capacities](https://github.com/CommonCoreOntology/CommonCoreOntologies/issues/229) issue, we seem to be coming closer to some definite thoughts

**Comment by neilotte** [comment]
@APCox When you have a moment, please see @alanruttenberg's request above.
